From bb2399bba9f048ee69a9213c1e2b43784ac40e47 Mon Sep 17 00:00:00 2001
From: Karl Otness <karl.otness@nyu.edu>
Date: Thu, 7 Sep 2023 18:06:01 -0400
Subject: [PATCH] Restore linker flags when testing for functions.

Omitting these causes detection for pthread builds to fail with static
libraries since the pthread flags are not passed to the linker.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index c7d3c092..7e9a3884 100755
--- a/setup.py
+++ b/setup.py
@@ -466,7 +466,7 @@ def has_function(self, function, includes=None, objects=None, libraries=None,
                             "a.out",
                             output_dir=tmpdir,
                             libraries=libraries,
-                            # extra_preargs=linker_flags,
+                            extra_preargs=linker_flags,
                             library_dirs=library_dirs,
                         )
 
